tests/push/test_ami_push.py

```diff
@@ -136,6 +147,42 @@ def test_do_push(command_tester, requests_mocker):
         ],
     )
 
+    # Check that aws publish has been called twice
```

Comment:
Why does the comment say twice but the next line of code asserts it was called once?